ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse.yaml

```diff
@@ -36,10 +36,10 @@ tests:
 - as: installer-rehearse-aws
   interval: 960h
   steps:
-    cluster_profile: aws-qe
+    cluster_profile: aws-c2s-qe
```

Comment:
I suggest to add a dedicated test job for each cluster profile, so that other subteam can easily test their own jobs.

Reply:
OK, will add them in another PR

Reply:
added profiles in https://github.com/openshift/release/pull/28339